src/test/java/me/nurio/events/eventdispathevent/EventDispatchEventCancellationTest.java

```diff
@@ -1,42 +1,43 @@
 package me.nurio.events.eventdispathevent;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import lombok.Setter;
 import me.nurio.events.EventManager;
-import me.nurio.events.handler.Event;
+import me.nurio.events.api.Event;
 import me.nurio.events.handler.EventDispatchEvent;
-import me.nurio.events.handler.EventHandler;
-import me.nurio.events.handler.EventListener;
+import me.nurio.events.annotation.EventHandler;
+import me.nurio.events.api.EventListener;
 import me.nurio.events.internal.ReflectedEventManager;
-import org.junit.Before;
-import org.junit.Test;
-
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 /**
- * This test will ensure that the EventDispatchEvent cancellation prevents the event from being dispatched.
+ * This test will ensure that the EventDispatchEvent cancellation prevents the event from being
+ * dispatched.
```

Comment:
Try to avoid editing lines that you don't need to update. The less you change, fewer things can cause chances of not getting merged.

This repeats all over the tests.